jvm/sargon-android/src/main/java/com/radixdlt/sargon/extensions/DappToWalletInteractionUnvalidated.kt

```diff
@@ -0,0 +1,13 @@
+package com.radixdlt.sargon.extensions
+
+import com.radixdlt.sargon.DappToWalletInteractionUnvalidated
+import com.radixdlt.sargon.dappToWalletInteractionUnvalidatedToJsonBytes
+import com.radixdlt.sargon.newDappToWalletInteractionUnvalidatedFromJsonBytes
+
+fun DappToWalletInteractionUnvalidated.Companion.fromJson(json: String) = runCatching {
+    newDappToWalletInteractionUnvalidatedFromJsonBytes(json.toByteArray().toBagOfBytes())
+}
+fun DappToWalletInteractionUnvalidated.toJson() = runCatching {
+    dappToWalletInteractionUnvalidatedToJsonBytes(this).string
+}
```

Comment:
We never exposed `Result` from kotlin extensions. We used to handle results in android and annotate these methods with `@Throws`. I would suggest keeping the same code style. Either expose result from all sargon methods or not.